base_multi_search_separator/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+* This module allows mulit_searching for all search.
```

Comment:
could you elaborate the description ? say exactly what the modules does.
(there are some base_search_xxx module in the OCA, like base_name_search_improved, base_search_fuzzy, etc...)